assets/push-images.sh

```diff
@@ -75,7 +75,7 @@ for i in "${linux_images[@]}"; do
         image_name="${reg}/${i}"
         ;;
     *)
-        image_name="${reg}/loftsh/${i}"
+        image_name="${reg}/ghcr.io/loft-sh/${i}"
```

Comment:
similar to my other comment, are we sure that "ghcr.io" part should be here? Will it work with all image registries?

Reply:
This is for consistency with the other images that hit the other case above. TBH, none of the images we list will match this regex since they all have a slash.